server/sonar-web/src/main/js/components/shared/drilldown-link.js

```diff
@@ -55,7 +55,9 @@ export class DrilldownLink extends React.PureComponent {
   propsToIssueParams = () => {
     const params = {};
 
-    if (this.props.periodDate) {
+    if (this.props.sinceLeakPeriod) {
```

Comment:
Ok, I thought it was only for quality gates conditions.
Then I'll also update `Coverage` and `Duplications`.
But there is something weird with those components, because they set the `period` property instead of `periodDate`...